packit_service/worker/handlers/github_handlers.py

```diff
@@ -468,6 +468,7 @@ def __init__(
         data: EventData,
         chroot: str,
     ):
+        data.trigger = TheJobTriggerType.pull_request
```

Comment:
How do we know it's a pull request? What about tests for branch push?

We should probably update `triggers` to support all three (release, PR, commit).

Reply:
OK, this [line](https://github.com/packit-service/packit-service/blob/17ccef68ecd5cc12282f6555ede80e18f5f80a7f/packit_service/worker/handlers/github_handlers.py#L462) confused me. 

Then forwarding the build model id should be better (and using then `build.job_trigger.get_trigger_object()`)

Reply:
> OK, this [line](https://github.com/packit-service/packit-service/blob/17ccef68ecd5cc12282f6555ede80e18f5f80a7f/packit_service/worker/handlers/github_handlers.py#L462) confused me.

So did me.